scripts/irods/test/test_all_rules.py

```diff
@@ -192,7 +192,6 @@ def filter_rulefiles(rulefile):
                 "rulemsiSetPublicUserOpr",
                 "rulemsiSetRandomScheme",
                 "rulemsiSetRescQuotaPolicy",
-                "rulemsiSetResource",
```

Comment:
Please check the icommands repo for references to that name. There's likely a rule file sharing that same name in that repo.

Reply:
I checked, and it does not seem to exist in the icommands repo.

Reply:
Good.